Tigran Kirakosov/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap" rel="stylesheet">
+  <title>Tokyo Weather Widget</title>
+</head>
+<body>
+  <article class="weather-container_main">
+    <div class="weather-container_widget">
+      <div class="weather-container_first-half">
+        <div class="current-weather__state"><img class="current-weather__state_image" src="/assets/images/icons/sun-ico.png"></div>
+        <div class="current-weather__desc">Sunny</div>
+        <div class="current-weather__date">Tokyo - Wednesday, July 31</div>
+      </div>
+      <div class="weather-container_second-half">
+        <div class="current-weather__temperature">28 &deg;C</div>
+        <div class="current-weather__graph"><img class="current-weather__graph_image" src="/assets/images/graph.png"></div>
+        <div class="current-weather__next-days">
+          <div class="day">
+            <img class="day_symb" src="/assets/images/icons/sunny.png"><div class="current-weather__day day-props">Thu</div><div class="day-props">28 &deg;C</div><div class="night-props">18 &deg;C</div>

```

Comment:
`<img>` tag has two required attributes - src and alt https://www.w3schools.com/tags/tag_img.asp

Reply:
So I should always keep these `alt` attributes in mind. Though it seemed to be unnecessary for me, and if that is required - so be it